v1/english/main.tex

```diff
@@ -0,0 +1,845 @@
+
+This change was made, in part, because Augur cannot\linebreak police offline trading. Shares of market outcomes are\linebreak simply tokens, which can be traded freely between users.\linebreak Since collecting fees on every trade is infeasible, Augur\linebreak instead collects fees only when traders settle directly with\linebreak the Augur market contracts. An added benefit of this ap-\linebreak proach is that it reduces the average fees paid by traders,\linebreak which should make Augur more competitive. \vspace*{-0.5\baselineskip}
+
+\subsection*{3. Universes}
+
+In the old design, there was only one ``version'' of REP,\linebreak and its total supply was fixed. In the current design, REP\linebreak can fork into many different versions (universes), each of\linebreak which can end up with more or less total REP than the\linebreak original version. If a fork is contentious, the REP supply\linebreak in each child universe might be only a fraction of the\linebreak total supply in the parent universe. In a non-contentious\linebreak fork, the early migration bonus to fork participants could\linebreak result in a child universe that has more total REP than\linebreak its parent universe.
+
+The new versions of REP spawned by a fork are all dif-\linebreak ferent tokens, each with its own price and total supply,\linebreak and service providers should treat them as such. When\linebreak Augur first launches, there will be a single universe (the\linebreak genesis universe) and a single version of REP, just as it\linebreak exists now. However, as soon as a fork occurs, the single\linebreak version of REP will split into many versions: for example,\linebreak a forking market with outcomes \texttt{A} and \texttt{B} would spawn new\linebreak tokens REP-\texttt{A}, REP-\texttt{B}, and REP-\texttt{Invalid}. Wallets and\linebreak exchanges that support REP would now have four differ-\linebreak ent versions of REP which they could (in theory) support\linebreak -- REP-genesis (the original version of REP, which would\linebreak now be locked), REP-\texttt{A}, REP-\texttt{B}, and REP-\texttt{Invalid}.\footnote{As a practical matter, service providers may find it easiest (and\linebreak least disruptive to their users) to encourage their users to partici-\linebreak pate in the fork, and then to simply support the winning universe\linebreak once the fork has resolved.}
+
+The total supply of REP in each child universe depends\linebreak on how much REP migrated to it, and when that migra-\linebreak tion occurred. Migrating REP during a fork, before it\linebreak is clear which child universe has achieved consensus, ex-\linebreak poses the user to a small (but non-zero) amount of risk\linebreak (see Section~\ref{section:Uncertain Fork Participation}), which may discourage participation\linebreak during the forking period of contentious forks. In order\linebreak to encourage participation during a fork, users must be\linebreak compensated for the risk.
```

Comment:
Hmm, this still isn't rendering even after the change that I thought would fix it.  Maybe it is a problem with the syntax or something?  I know very little about LaTeX, so I'm not sure how to troubleshoot it further.
![{C47AB74A-A2FF-4203-BA2B-9DCB5785E76E}](https://github.com/user-attachments/assets/937a5369-b1d3-43fd-a0c1-11a2d916ae50)